docs/content/installation/metrics/_index.md

```diff
@@ -0,0 +1,22 @@
+---
+title: "PostgreSQL Operator Metrics"
```

Comment:
Let's all this "PostgreSQL Operator Monitoring" or "PostgreSQL Operator - Monitoring with pgMonitor"

Reply:
Just to clarify, do we want to switch to "PostgreSQL Operator Monitoring" across the board, i.e. anywhere "PostgreSQL Operator Metrics" is referenced.  "PostgreSQL Operator Metrics" is scattered all over throughout the docs, so just wanted to verify before making this change.

Reply:
I went ahead and made a wholesale change to the docs (and a couple of other files) from "PostgreSQL Operator Metrics" to "PostgreSQL Operator Monitoring" (in addition to a few other general changes from "metrics" to "monitoring").  All renames are included in [this commit](https://github.com/CrunchyData/postgres-operator/pull/1757/commits/e476821b3cc6c8fefcb9d2108bbaa51e2c31e1bd).  Also, please note that various directories, files and certain places in the docs (where applicable) as included in this PR still specify "metrics".